coverage.md

```diff
@@ -35,14 +35,15 @@ Manual Coverage
 | **Copy** |  |  |
 | As a user, I can copy any content by adding it to a repository with modify/ endpoint (but nothing is copied automatically, and invalid repositories will fail to validate for some definition of “invalid”) | NO |  |
 | As a user, I can copy any content by href using Copy API | PART |  |
-| As a user, I can copy RPM package and its dependencies (if depsolving=True) | Yes | to empty and non-empty repository |
+| As a user, I can copy RPM package and its dependencies (if depsolving=True) | YES | to empty and non-empty repository |
 | As a user, I can copy Advisory and packages it refers to (and their dependencies if depsolving=True) by copying the Advisory | PART |  |
 | As a user, I can copy PackageCategories, PackageEnvironments and PackageGroups (and their dependencies) | PART | no packageenvironment test |
 | As a user, I can copy Modulemd and its artifacts by copying the Modulemd | NO |  |
 | As a user, I can copy Modulemd with its artifacts and its module dependencies and artifacts’ dependencies (if depsolving=True). | NO |  |
 | As a user, if the default Modulemd is copied, its module-default is copied as well (and vice-versa) | NO |  |
 | As a user, I can copy content with dep solving on and specify multiple repositories to copy from/to | NO |  |
 | As a user, all content that I directly specify to be copied should always be copied (obviously, but we need to test it, there have been dependency solving bugs where it didn’t happen for various reasons) | NO |  |
+| Dependencies can be solved for RPM packages which depend on specific files (such as /usr/bin/bash) present only in filelists.xml | NO | needs a fixture change/improvement |
```

Comment:
I think I remember a document (hackmd or google docs) where we were documenting things that we needed to improve in the fixtures, but I can't find it.

Reply:
Hmm, I'm not aware of such or forgot.
Only the one which @pavelpicka  and I brainstormed for the distribution trees.